config/plans.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-env node */
+'use strict';
+
+const monthly = [
+  {builds: 1, price: 6900, currency: 'USD', name: 'Bootstrap', description: 'Ideal for hobby projects'},
+  {builds: 2, price: 12900, currency: 'USD', name: 'Startup', description: 'Best for small teams'},
+  {builds: 5, price: 24900, currency: 'USD', name: 'Small Business', description: 'Great for growing teams'},
+  {builds: 10, price: 48900, currency: 'USD', name: 'Premium', description: 'Perfect for larger teams'}
+];
+
+const annual = [
+  {builds: 1, price: 75900, currency: 'USD', name: 'Bootstrap', description: 'Ideal for hobby projects'},
+  {builds: 2, price: 141900, currency: 'USD', name: 'Startup', description: 'Best for small teams'},
+  {builds: 5, price: 273900, currency: 'USD', name: 'Small Business', description: 'Great for growing teams'},
+  {builds: 10, price: 537900, currency: 'USD', name: 'Premium', description: 'Perfect for larger teams'}
+];
+
+monthly.forEach(plan => {
+  plan.monthlyPrice = plan.price;
+  plan.monthlyPriceRounded = Math.round(plan.monthlyPrice / 100);
+});
+
+annual.forEach(plan => {
+  plan.monthlyPrice = plan.price / 12;
+  plan.monthlyPriceRounded = Math.round(plan.monthlyPrice / 100);
+});
+
+module.exports = { annual, monthly };
```

Comment:
I think it does make sense. Maybe the best way to approach this is to split this bit off into its own tiny PR, that we can make to fit both needs, merge into `master`, and then pull into our respective PRs. What do you think @emmanuel-ci @artursmirnov ?

Reply:
Splitting it to a separate PR sounds like a great idea and makes a lot of sense 👍 Though, I'd suggest to keep them all in one list, setting period as a property:
```js
module.exports = {
  plans: [
    {
      builds: 1,
      price: 75900,
      period: 'monthly',
      name: 'Bootstrap',
      isEnabled: true,
      isDefault: false,
      ...
    }
  ]
}
```
That would allow to derive any other subsets dynamically:
```js
import config from 'travis/config/environment'
const { plans } = config;
const monthly = plans.findBy('period', 'monthly');
```

What do you think?

Reply:
I think having a single list with `period` as a property makes a lot of sense, I like it :+1: 

As for getting the price etc from the backend, I like it, but I think it may need some adjustments to the api. The endpoint that I found doesn't seem geared towards querying for specific plans or types of plans. But maybe it's worth looking into.

Reply:
Ok, that sounds good to me :) Would you like to make the plans config PR, or shall I? Either way works for me

Reply:
Yeah that's perfectly fine :+1: Have a good rest of your day!